.circleci/regenerate.py

```diff
@@ -251,6 +251,30 @@ def cmake_workflows(indentation=6):
             jobs.append({f'cmake_{os_type}_{device}': job})
     return indent(indentation, jobs)
 
+def ios_workflows(indentation=6):
+    jobs = []
+    build_job_names = []
+    for arch, platform in [('x86_64', 'SIMULATOR'), ('arm64', 'OS')]:
+        name = f'libtorchvision_ops_ios_12.0.0_nightly_{arch}_build'
+        build_job_names.append(name)
+        build_job = {
+            'build_environment': f'libtorchvision_ops-ios-12.0.0-nightly-{arch}-build',
+            'context': 'org-member',
+            'filters': gen_filter_branch_tree('nightly'),
```

Comment:
FYI I believe the jobs are not currently running in the PR because you filtered it only to the nightly branch. If you remove this filter the job should run in your PR I believe